JDI.Light/JDI.Light/Factories/WebDriverFactory.cs

```diff
@@ -155,19 +155,9 @@ public string RegisterDriver(string driverName)
 
         public void SetRunType(string runType)
         {
-            switch (runType)
-            {
-                case "local":
-                    RunType = RunType.Local;
-                    return;
-                case "remote":
-                    RunType = RunType.Remote;
-                    return;
-            }
 
-            RunType = RunType.Local;
+            if (runType == "local")? RunType = RunType.Local : RunType = RunType.Remote;
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5844034)